client/my-sites/customer-home/components/full-screen-launchpad.tsx

```diff
@@ -59,41 +73,63 @@ export const FullScreenLaunchpad = ( { onClose }: { onClose: () => void } ): JSX
 		dispatch( requestSite( siteId ) );
 	};
 
+	const {
+		data: { checklist },
+	} = useSortedLaunchpadTasks( siteSlug, checklistSlug, launchpadContext );
+
 	if ( isDismissed ) {
 		return null;
 	}
 
-	const isAllTasksCompleted = hasChecklist && numberOfSteps > 0 && completedSteps === numberOfSteps;
+	const launchSiteTask = checklist?.find( ( task: Task ) => task.isLaunchTask );
+
+	const isAllTasksCompleted =
+		hasChecklist &&
+		numberOfSteps > 0 &&
+		completedSteps >= numberOfSteps - ( launchSiteTask ? 1 : 0 );
 
 	return (
-		<div css={ { display: 'flex', flexDirection: 'column', alignItems: 'center' } }>
-			<div className="is-launchpad-first" css={ { width: '100%' } }>
-				<div className="customer-home-launchpad customer-home__card is-small-hero">
-					<div className="customer-home__launchpad-header">
-						<CircularProgressBar
-							size={ 40 }
-							enableDesktopScaling
-							numberOfSteps={ numberOfSteps }
-							currentStep={ completedSteps }
-						/>
-						<h2>
-							{ ! isAllTasksCompleted ? __( "Let's get started!" ) : __( "You're all set!" ) }
-						</h2>
-						<span>{ ! isAllTasksCompleted && launchpadTitle }</span>
-					</div>
-					{ isAllTasksCompleted && <ConfettiAnimation /> }
-					<Launchpad
-						siteSlug={ siteSlug }
-						checklistSlug={ checklistSlug }
-						launchpadContext={ launchpadContext }
-						onSiteLaunched={ onSiteLaunched }
-						highlightNextAction
+		<div className="is-launchpad-first" css={ { width: '100%' } }>
+			<div
+				className={ `customer-home-launchpad customer-home__card is-small-hero ${
+					isAllTasksCompleted ? 'all-tasks-completed' : ''
+				}` }
+			>
+				<div className="customer-home__launchpad-header">
+					<CircularProgressBar
+						size={ 40 }
+						enableDesktopScaling
+						numberOfSteps={ numberOfSteps - ( launchSiteTask ? 1 : 0 ) }
+						currentStep={ completedSteps }
```

Comment:
Really polishing now, but we should also subtract 1 from `completedSteps` here if one of the completed steps is the launch step. In the few milliseconds during which the site has finished launching, but we haven't finished redirected yet, the progress donut renders `6 / 5`.


https://github.com/user-attachments/assets/e8ec3884-1531-4d1b-b465-af67599d4ce6

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/99563/commits/b16a99aa49d342a8fbd23401dbcd2adc24d03f7f

Reply:
Good catch!